app/konnect-compatibility.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "{{site.konnect_short_name}} compatibility"
+description: 'Details which browsers, software, and versions {{site.konnect_short_name}} is compatible with.'
+content_type: reference
+layout: policy
```

Comment:
```suggestion
content_type: policy
layout: reference
```
Sorry, I accidentally commented on the layout not the content type originally. Should be this way.